store/store.go

```diff
@@ -466,6 +470,10 @@ func (bs *BlockStore) SaveTxInfo(block *types.Block, txResponseCodes []uint32) e
 			Index:  uint32(i),
 			Code:   txResponseCodes[i],
 		}
+		// Only add rawLog for failed transactions
+		if txResponseCodes[i] != 0 {
```

Comment:
would put `abci.CodeTypeOk` instead of relying on numbers

Reply:
done